src/style.css

```diff
@@ -1 +1,125 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir-regular;
+  src: url(fonts/Avenir.ttc);
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: Avenir-book;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 700;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Avenir-book, serif;
+  position: relative;
+}
+
```

Comment:
rewrite all your styles to new html naming + try to reuse styles where it's possible (don't repeat yourself)

copy/past example:

[data-qa='big'] {
  margin-top: 20px;
  position: relative;
}
[data-qa='small'] {
  margin-top: 20px;
  position: relative;
}